osx-utils/performous-app-build.sh

```diff
@@ -331,7 +331,7 @@ function main {
 			  -DCMAKE_CXX_COMPILER="${CXXPATH}" \
 			  -DCMAKE_EXPORT_COMPILE_COMMANDS="ON" \
 			  -DCMAKE_CXX_FLAGS="-Wall -Wextra" \
-			  -DCMAKE_OSX_ARCHITECTURES="x86_64" \
+			  -DCMAKE_OSX_ARCHITECTURES=${CMAKE_HOST_SYSTEM_PROCESSOR} \
```

Comment:
This is actually incorrect. At this point we're in bash, so you 're passing in an empty string.
OTOH, I don't really think that argument is necessary at all nowadays, so perhaps you could remove it entirely.

Reply:
Oh dang, I guess I misunderstood what was happening. That said, you're right that removing the flag entirely works too. I've checked that in.

Reply:
I've also added you as a collaborator on my fork so you can mess with my branch if you like.